core/templates/dev/head/components/forms/FormBuilder.js

```diff
@@ -935,7 +935,19 @@ oppia.directive('textAngularRte', [
           };
 
           $scope.stripFormatting = function(html) {
-            return $filter('sanitizeHtmlForRte')(html);
+            var whitelistedTags = [
```

Comment:
Instead of hard-coding this, derive the list from analyzing `rteHelperService.getRichTextComponents()`, so that it will be updated if we add new components in the future. Also, define whitelistedTags outside this method, so that it is computed just once.